ckanext/example_iauthfunctions/tests/test_example_iauthfunctions.py

```diff
@@ -260,18 +272,33 @@ def test_group_create_with_curator(self):
         assert result['name'] == name
 
 
-class TestExampleIAuthFunctionsPluginV3(TestExampleIAuthFunctionsPluginV4):
+class TestExampleIAuthFunctionsPluginV3(BaseTest):
     '''Tests for the ckanext.example_iauthfunctions.plugin_v3 module.
 
     '''
     @classmethod
     def setup_class(cls):
-        cls.app = paste.fixture.TestApp(pylons.test.pylonsapp)
-        ckan.plugins.load('example_iauthfunctions_v3')
+        '''Nose runs this method once to setup our test class.'''
+
+        # Test code should use CKAN's plugins.load() function to load plugins
+        # to be tested.
+        if not ckan.plugins.plugin_loaded('example_iauthfunctions_v3'):
+            ckan.plugins.load('example_iauthfunctions_v3')
+
+        # Make a copy of the Pylons config, so we can restore it in teardown.
+        cls._original_config = dict(config)
+        config['ckan.plugins'] = 'example_iauthfunctions_v3'
+
+        # Make the TestApp that we'll use to simulate HTTP requests to
+        # CKAN.
+        cls.app = helpers._get_test_app()
 
     @classmethod
     def teardown_class(cls):
         ckan.plugins.unload('example_iauthfunctions_v3')
+        # Restore the config to its original values
+        config.clear()
+        config.update(cls._original_config)
```

Comment:
can we use `FunctionalTestBase`'s `_load_plugins` class variable instead of this boilerplate?